internal/scrubber/scrub_sensitive_test.go

```diff
@@ -12,7 +12,11 @@ import (
 type S struct {
 	SomePassword1 string `kopia:"sensitive"`
 	NonPassword   string
-	Inner         *Q
+	InnerPtr      *Q
```

Comment:
Does extending the struct cover the desired cases? or should the text be extended with additional "input cases"?

Reply:
I initially started writing a table driven test with inputs and expected outputs, but concluded it was a lot of boilerplate to accomplish the same conditions: instead of defining different test structs for the different test cases we could get the same coverage by adding the newly-handled field types in the existing test struct.